packages/api-reference/src/components/Content/Lazy/Loading.vue

```diff
@@ -162,7 +166,13 @@ onMounted(() => {
         <Operation
           v-for="operation in tag.lazyOperations"
           :key="`${operation.httpVerb}-${operation.operationId}`"
+          :collection="activeCollection"
+          :layout="layout"
           :operation="operation"
+          :requestExamples="requestExamples"
+          :requests="requests"
+          :securitySchemes="securitySchemes"
+          :server="activeServer"
```

Comment:
so many props! should we use the store instead?

Reply:
we will discuss in paris!